infrastructure/aws/cdk/app.py

```diff
@@ -325,65 +215,53 @@ def __init__(  # noqa: C901
                 env["DB_MAX_CONN_SIZE"] = "1"
             if "DB_MIN_CONN_SIZE" not in env:
                 env["DB_MIN_CONN_SIZE"] = "1"
-
-            eostac_function = aws_lambda.Function(
-                self,
-                f"{id}-stac-lambda",
-                runtime=aws_lambda.Runtime.PYTHON_3_11,
-                code=aws_lambda.Code.from_docker_build(
-                    path=os.path.abspath(context_dir),
-                    file="infrastructure/aws/dockerfiles/Dockerfile.stac",
-                    build_args={
-                        "PYTHON_VERSION": "3.11",
-                    },
-                    platform="linux/amd64",
-                ),
-                vpc=vpc,
-                handler="handler.handler",
-                memory_size=eostac_settings.memory,
-                timeout=Duration.seconds(eostac_settings.timeout),
-                environment=env,
-                log_retention=logs.RetentionDays.ONE_WEEK,
-            )
-            for k, v in db_secrets.items():
-                eostac_function.add_environment(key=k, value=str(v))
-
+            env.update(db_secrets)
             # If raster is deployed we had the TITILER_ENDPOINT env to add the Proxy extension
```

Comment:
vincent do you have a doc link for this proxy extension ? haven't read about it and i'm curious !

Reply:
it's not really a proxy. But for our custom eoAPI runtime we add a titiler extension https://github.com/developmentseed/eoAPI/blob/main/runtime/eoapi/stac/eoapi/stac/extension.py which is used to add a `/collections/{collectionId}/items/{itemId}/tilejson.json` endpoint to the STAC service which will redirect to the raster tilejson endpoint. This is really custom for our custom runtime